app/graphql/mutations/trip_create.rb

```diff
@@ -10,15 +10,18 @@ class TripCreate < BaseMutation
     field :trip, Types::TripType, null: false
 
     def resolve(name:, category:, traveler:)
-      # trip = ::Trip.new(name: name, category: category)
       trip = Trip.new(name: name, category: category, traveler: traveler, image: image_selector(category))
       raise GraphQL::ExecutionError.new "Error creating trip", extensions: trip.errors.to_hash unless trip.save
       { trip: trip }
     end
 
     def image_selector(category)
-      "https://www.sandals.com/blog/content/images/2020/03/Sandals-Barbados-Beach-Palm-Trees-Beach.jpg" if category == 2
-      "https://www.lonestar.edu/images/internationalTravel.jpg" if category == 4
+      return "https://www.sandals.com/blog/content/images/2020/03/Sandals-Barbados-Beach-Palm-Trees-Beach.jpg" if category == "beach"
```

Comment:
What if you created a hash where the key is the category and the value is the image affiliated with the image? That way, you can just have one return statement
`return image_for_category[category]` if image_for_category is your hash of images? You would then just need one fallback image if the given category doesn't exist in the hash